src/main/java/net/sf/jabref/gui/shared/OpenSharedDatabaseDialog.java

```diff
@@ -198,6 +207,16 @@ private void applyGlobalPrefs() {
         if (sharedDatabaseUser.isPresent()) {
             userField.setText(sharedDatabaseUser.get());
         }
+
+        if (sharedDatabasePassword.isPresent() && sharedDatabaseUser.isPresent()) {
+            try {
+                passwordField.setText(new Password(sharedDatabasePassword.get(), sharedDatabaseUser.get()).decrypt());
+            } catch (GeneralSecurityException e) {
+                LOGGER.error("Could not read the password due to decryption problems.");
```

Comment:
LOGGER.errror has a second paramter for the exception, so just pass, `e` as 2nd parameter